layersvt/device_simulation.cpp

```diff
@@ -48,6 +48,8 @@
 #include <fstream>
 #include <mutex>
 
+using namespace std;
```

Comment:
Got it.  Will use atoi() for Android and std::atoi() for none Android explicitly.